INSTALLATION.md

```
@@ -0,0 +1,255 @@
+## Shell Configuration
+
+### Single Source of Truth
+
+All shell configurations are generated from `scripts/nu/setup-local-machine/config.toml`:
+
+```toml
+[aliases]
+lg = "lazygit"
+k = "kubectl"
+
+[functions]
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)